rust/Cargo.toml

```diff
@@ -7,7 +7,7 @@ categories = [ "science::robotics", "compression" ]
 repository = "https://github.com/foxglove/mcap"
 documentation = "https://docs.rs/mcap"
 readme = "README.md"
-version = "0.7.0"
+version = "0.7.1"
```

Comment:
Will we need to do anything to release this version to crates?

Reply:
The only previous instance of a "release" I've found is this tag: https://github.com/foxglove/mcap/releases/tag/releases%2Frust%2Fv0.6.0

Reply:
@james-rms is the owner of the crate (see https://crates.io/crates/mcap) so he'll have to push an update with [`cargo publish`](https://doc.rust-lang.org/cargo/reference/publishing.html) unless someone has his credentials sitting around. :stuck_out_tongue: